forge/src/main/java/org/openjdk/skara/forge/github/GitHubRepository.java

```diff
@@ -582,4 +582,37 @@ public void restrictPushAccess(Branch branch, HostUser user) {
                       .map(o -> new Label(o.get("name").asString(), o.get("description").asString()))
                       .collect(Collectors.toList());
     }
+
+    @Override
+    public void addLabel(Label label) {
+        var params = JSON.object()
+                .put("name", label.name())
+                // Color is Gray and matches all current labels
+                .put("color", "ededed");
```

Comment:
Will this overwrite the existing color scheme? For a new repo, will it always use gray for all labels?

Reply:
If addLabel() is called on an existing label, then I think the color scheme will be overwritten. The intention not to change any label, just to add the ones that are missing. The updateLabel method will not change the color.

Note that the labels we are adding with this patch are just the ones for mailing lists. Do you think we need to add automation for colors for the other set of labels we use (e.g. rfr, ready, csr etc)?

Reply:
I think if we do add automation for colors for the other set of labels, that could be done as a follow-on fix (not sure it's needed though). I mainly wanted to make sure we wouldn't overwrite the once for rfr, ready, etc. with gray.